tests/playwright-test/runner.spec.ts

```diff
@@ -449,3 +449,18 @@ test('should not crash with duplicate titles and line filter', async ({ runInlin
     `   - example.spec.ts:8`,
   ].join('\n'));
 });
+
+test('should allow wildcards in project option', async ({ runInlineTest }) => {
+  const result = await runInlineTest({
+    'playwright.config.ts': `
+    module.exports = { name: 'project-name' };
+    `,
+    'a.test.js': `
+      const { test } = pwt;
+      test('one', async ({}) => {
+        expect(1).toBe(1);
+      });    `
+  }, { project: '.*oj.*t-na.?e' }, {}, {});
```

Comment:
Let's also add a test where one of the filters doesn't match any projects.

Reply:
It can be something similar to [this test](https://github.com/microsoft/playwright/blob/76cffad023cc01f5f85d16b05d48cc32cd7fa302/tests/playwright-test/runner.spec.ts#L31-L32) which would assert the output contains expected error message.